pkg/cmd/login/browser.go

```diff
@@ -47,7 +47,12 @@ func (h *Handler) handle() http.Handler {
 		}
 
 		ctx := r.Context()
-		u, err := okteto.Auth(ctx, code, h.baseURL)
+		oktetoClient, err := okteto.NewOktetoClientFromUrl(h.baseURL)
```

Comment:
I think func `NewOktetoClientFromUrl` has enough logic to add some tests for it? :)

Reply:
NIT: I see this func was added before your changes

Reply:
That should be addressed when we do the refactor for the `client.go`